tests/integration/components/tooltip-is-visible-test.js

```diff
@@ -1,6 +1,6 @@
 import Ember from 'ember';
 import { moduleForComponent, test } from 'ember-qunit';
-import { assertHide, assertShow } from '../../helpers/sync/assert-visibility';
+import { assertHide, assertShow, assertTetherEnabled, assertTetherDisabled } from '../../helpers/sync/assert-visibility';
```

Comment:
Yeah, good point. I'll try split it (or make a new PR if you want to merge already).